### PR TITLE
Document SEO error pages

### LINKED ISSUE #606: [SeoBundle][error handling] Docs for error handling
... just a reminder.

Feature was added in https://github.com/symfony-cmf/SeoBundle/pull/192
and updated in https://github.com/symfony-cmf/SeoBundle/pull/251

### PR DESCRIPTION
**Pull request by WouterJ**
Fixes #606 

/cc @ElectricMaxxx

### CODE REVIEW

File: bundles/seo/error_pages.rst [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+beginning).
+
+The CmfSeoBundle provides an error controller that does exactly this. By using
+suggestion providers, the controller finds the most relevant pages and shows
+this on the error page.
+
+Using the Exception Controller
+------------------------------
+
+To use the exception controller provided by this bundle, configure the twig
```
Comment (ElectricMaxxx) [inline bundles/seo/error_pages.rst]: Maybe we should explain the jump from the `SuggestionProviderController` to the `ErrorController`. Yea, the `SuggestionProviderController` is an replacement for the common twig error controller. So we should add at least an link to the symfony/twig documentation, right?

File: bundles/seo/error_pages.rst [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+comes with two built-in providers:
+
+``ParentSuggestionProvider``
+    This provides the parent page of the not found page.
+``SiblingSuggestionProvider``
+    This provides the siblings of the current page.
+
+.. note::
+
+    Both providers only work with PHPCR documents.
```
Comment (ElectricMaxxx) [inline bundles/seo/error_pages.rst]: Can we add and an example what it means to get an parent/sibbling route for an 404 on a given route?

**Comment by ElectricMaxxx** [comment]
Had two little issues only, but thanks for doing that documenation. its great.

File: bundles/seo/twig.rst [hunk 3]
```diff
@@ -35,6 +35,7 @@ base twig template and override it in a sub template, calling
     {% extends 'base.html.twig' %}
 
     {% block metadata %}
-        {{ cmf_seo_update_metadata(post) }}
+        {% do cmf_seo_update_metadata(post) %}
```
Comment (dbu) [inline bundles/seo/twig.rst]: oh, is this the twig syntax for calling something from twig for the side effect and not to output the result?

Comment (WouterJ) [inline bundles/seo/twig.rst]: Exactly

**Comment by dbu** [comment]
great, i let you and max wrap this up, looks good from my point of view.

File: bundles/seo/error_pages.rst (hunk 1 above)
Comment (WouterJ) [inline bundles/seo/error_pages.rst]: added

File: bundles/seo/error_pages.rst (hunk 2 above)
Comment (WouterJ) [inline bundles/seo/error_pages.rst]: done

**Pull request merged**